package student_yevgeniy_tolks.lesson_13_exceptions.level_4.task23_24.task26;

class StackOverflowErrorTest {

// current class pops-up StackOverflowError using recursion

//    static int number = 0;
//
//    public static int printSomeNumber(int number1) throws Exception {
//        number += number1;
//        System.out.println(number);
//        return number1 + printSomeNumber(number + 1);
//    }
//
//    public static void main(String[] args) throws Exception {
//        System.out.println(printSomeNumber(number));
//    }
}
